### PR TITLE
Feature fortify refactor helper

### PR DESCRIPTION
**Pull request by hayleyyan**
REFACTOR:
- Clean up helper.py in fortfiy
    - moved download_scan -> download.py
    - list_application_version -> list.py
    - upload_scan -> upload.py 
- fix a mispelled file (main_fortify_click_test)

TEST:
- changed test_fortify_download_download_successful_download (not sure if I like the naming convention on the test file...)
- All test pass

### CODE REVIEW

File: tests/fortify/download_test.py [hunk 1]
```diff
@@ -111,7 +110,7 @@ def test_fortify_download_download_throws_attribute_error(log_mock, config_mock,
                         application_name=expected_application,
                         version_name=expected_version)
 
-    log_mock.assert_called_once()
```
Comment (matt-fevold) [inline tests/fortify/download_test.py]: did this break something? 
could change to log_mock.call_count == 1 if that's what you care about

File: tests/fortify/download_test.py [hunk 2]
```diff
@@ -133,7 +132,7 @@ def test_fortify_download_download_throws_unbound_local_error(log_mock, config_m
...
```
Comment (matt-fevold) [inline tests/fortify/download_test.py]: same issue.

File: tests/fortify/download_test.py [hunk 3]
```diff
@@ -155,4 +154,4 @@ def test_fortify_download_download_throws_io_error(log_mock, config_mock, auth_m
...
```
Comment (matt-fevold) [inline tests/fortify/download_test.py]: same issue

File: tests/fortify/download_test.py (hunk 1 above)
Comment (HomerGaidarski) [reply tests/fortify/download_test.py]: assert log_mock.call_count == 1

**Comment by matt-fevold** [comment]
I'd also rename the "test_fortify_download_download_successful_download" test. 

probably one download is necessary :D maybe "test_fortify_download_successful" ?

**Pull request merged**